client/my-sites/importer/newsletter/subscribers/paid-subscribers/map-plan.tsx

```diff
@@ -31,18 +33,18 @@ type MapPlanProps = {
 	selectedProductId: string;
 };
 
-function displayProduct( product?: Product ) {
```

Comment:
Simplified by turning into a component which only outputs plan details, not "select" string.

Reply:
Removed the change from this PR.